src/lib/features/playback-controls/ui/scrubbing-bar.svelte

```diff
@@ -0,0 +1,161 @@
+<script lang="ts">
+  import { formatDuration } from '$lib/shared/ui';
+
+  import { createEventDispatcher } from 'svelte';
+
+  export let duration: number;
+  export let position: number;
+  // export let buffered: TimeRanges;
```

Comment:
Is this commented-out code left here intentionally?

Reply:
Yes, I've been meaning to come back to it at some point to render the buffered ranges.